src/parser/test/parser_test.py

```diff
@@ -93,3 +93,78 @@ def test_nested_block_parser():
         assert expected[idx].content() == cur[0].content()
         cur = cur[0].children
         idx += 1
+
+
+def test_parse_md_to_ast_simple():
+    md = 'abc***bold***'
+    root = parse_md_to_ast(md)
+    assert isinstance(root, Element)
+    assert len(root.children) == 1
+    assert isinstance(root.children[0], TextParagraph)
+    assert isinstance(root.children[0].children[0], TextBlock)
+    assert isinstance(root.children[0].children[1], BoldBlock)
+    assert isinstance(root.children[0].children[1].children[0], ItalicBlock)
+    assert root.children[0].children[1].children[0].children[0].content() == 'bold'
```

Comment:
should have TextBlock inside ItalicBlock?

Reply:
right it's omitted in the test as the last one asserts the TextBlock's content